tests/index.test.js

```diff
@@ -189,7 +225,7 @@ it('can interpret various variations in specifying browser names', () => {
   expect(matchesUA(ua.safari.iOS('10.3.0'), {browsers: ['ios_saf >= 10.1.0']}))
     .toBeTruthy()
 
-  expect(matchesUA(ua.safari('10.3.0'), {browsers: ['ios_saf >= 10.1.0']}))
```

Comment:
From what I can tell, this test was just testing completely incorrect behavior. `ua.safari()` creates desktop useragent strings, not iOS ones. And also, Safari 10.3.0 is not a real browser version.